lapis2/src/main/kotlin/org/genspectrum/lapis/config/DatabaseConfig.kt

```diff
@@ -4,11 +4,17 @@ data class DatabaseConfig(val schema: DatabaseSchema)
 
 data class DatabaseSchema(
     val instanceName: String,
+    val opennessLevel: OpennessLevel,
     val metadata: List<DatabaseMetadata>,
     val primaryKey: String,
     val features: List<DatabaseFeature> = emptyList(),
 )
 
-data class DatabaseMetadata(val name: String, val type: String)
+data class DatabaseMetadata(val name: String, val type: String, val unique: Boolean = false)
 
 data class DatabaseFeature(val name: String)
+
+enum class OpennessLevel {
+    OPEN,
+    GISAID,
```

Comment:
I think that it's better if we name it `PROTECTED` (and should we later have different protected modes, we should try to find general names).

Also, would this be a good place to document briefly what the two levels mean?

Reply:
done